flexget/api.py

```diff
@@ -22,41 +25,111 @@ def attach_schema(response):
     return response
 
 
-@api_schema.route('/version')
-def version_schema():
-    return jsonify({
-        'type': 'object',
-        'properties': {
-            'flexget_version': {'type': 'string', 'description': 'FlexGet version string'},
-            'api_version': {'type': 'integer', 'description': 'Version of the json api'}
-        }
-    })
-
-
-@api.route('/version')
+@api.route('/version/')
 def version():
     return jsonify(flexget_version=flexget.__version__, api_version=API_VERSION)
 
 
+# Execution API
+
 exec_parser = get_parser('execute')
 
+def _get_task(task_id):
```

Comment:
@gazpachoking see my latest commit. As per our discussions task_queue now stores task data for at least an hr.

Still need to add the task_id to the log file for streaming to work.